### PR TITLE
[zh]Update other pages for links with '/zh/' prefix, using new prefix '/zh-cn/'

### PR DESCRIPTION
**Pull request by howieyuen**
relate issue: https://github.com/kubernetes/website/issues/34501, task-21

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -539,7 +539,7 @@
 /id/docs/setup/minikube/     /id/docs/tasks/tools/ 302
 /docs/setup/learning-environment/     /docs/tasks/tools/  302!
 /id/docs/setup/learning-environment/     /id/docs/tasks/tools/  302!
-/zh/docs/setup/learning-environment/     /zh-cn/docs/tasks/tools/  302!
+/zh-cn/docs/setup/learning-environment/     /zh-cn/docs/tasks/tools/  302!
```
Comment (tengqm) [inline static/_redirects:542]: This looks unnecessary.

File: static/_redirects [hunk 2]
```diff
@@ -218,7 +218,7 @@
 /docs/reference/glossary/maintainer/    /docs/reference/glossary/approver/ 301
 
 /docs/reference/kubectl/overview/    /docs/reference/kubectl/ 301
-/zh/docs/reference/kubectl/overview/    /zh-cn/docs/reference/kubectl/ 301
+/zh-cn/docs/reference/kubectl/overview/    /zh-cn/docs/reference/kubectl/ 301
```
Comment (tengqm) [inline static/_redirects:221]: This looks unnecessary.

File: static/_redirects [hunk 3]
```diff
@@ -18,7 +18,7 @@
 /ru/docs/     /ru/docs/home/ 301!
 /uk/docs/     /uk/docs/home/ 301!
 /vi/docs/     /vi/docs/home/ 301!
-/zh/docs/     /zh-cn/docs/home/ 301!
+/zh-cn/docs/     /zh-cn/docs/home/ 301!
```
Comment (tengqm) [inline static/_redirects:21]: This change is wrong. Please revert.

File: static/_redirects [hunk 4]
```diff
@@ -581,4 +581,4 @@
 /id/docs/concepts/configuration/pod-overhead/     /id/docs/concepts/scheduling-eviction/pod-overhead/   301
 
 /pt/*         /pt-br/:splat  302!
-/zh/*         /zh-cn/:splat  302!
+/zh-cn/*         /zh-cn/:splat  302!
```
Comment (tengqm) [inline static/_redirects:584]: This change is wrong. Please revert.

File: static/_redirects (hunk 1 above)
Comment (howieyuen) [reply static/_redirects:542]: Thanks to pointing

**Comment by howieyuen** [comment]
already rebase the latest commits on main branch

**Comment by ydFu** [comment]
I don't think the files covered by this PR are all in line with the issue situation this time.
update existing pages for links with '/zh/' prefix, using new prefix '/zh-cn/',
I think these files are more suitable to be pulled out for additional processing.
```
content/en/blog/_posts/2019-04-26-latest-on-localization.md
data/i18n/zh-cn/zh-cn.toml
scripts/README.md
scripts/linkchecker.py
scripts/lsync.sh
static/_redirects
```

**Pull request merged**